.idea/compiler.xml

```diff
@@ -20,10 +20,10 @@
       </profile>
     </annotationProcessing>
     <bytecodeTargetLevel>
-      <module name="killbill-base-plugin" target="1.6" />
+      <module name="killbill-base-plugin" target="1.8" />
```

Comment:
> Does it mean that any java plugin -- once moved to this `plugin-framework` version will need need to upgrade java `1.8`.

Yup.

> I think this is Ok, but for a long time i thought you were actually quite cautious to not do that?

I was -- and still am -- for the core (and our dependencies). For plugins, I care less and we have actually been requiring implicitly 1.7 for a while (at least for Adyen).